src/Inspector/Controller/OpcacheController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Debug\Api\Inspector\Controller;
+
+use Psr\Http\Message\ResponseInterface;
+use Yiisoft\DataResponse\DataResponseFactoryInterface;
+
+use function Safe\opcache_get_status;
+
+final class OpcacheController
+{
+    public function __construct(
+        private DataResponseFactoryInterface $responseFactory,
+    ) {
+    }
+
+    public function index(): ResponseInterface
+    {
+        return $this->responseFactory->createResponse([
+            'status' => opcache_get_status(true),
```

Comment:
Need to verify that opcache is available.

Reply:
Good idea. Or else it would just crash.